pkgs/applications/office/espanso/default.nix

```diff
@@ -55,10 +55,6 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
-  cargoPatches = lib.optionals stdenv.isDarwin [
```

Comment:
Is this the reason that the build fails on darwin? I noticed that this was only originally tested on x86_64-linux, but this is a darwin patch, so this should have been built or tested on darwin without this before commiting. Could someone test if re-adding this fixes the darwin build? I do not have a darwin machine myself so I cannot test this.

Reply:
```
>   ld: framework not found System
       >   clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
       >   make[1]: *** [Makefile:117: wxrc] Error 1
```

Reply:
yeah i think its this patch, I can test later today